.NET/Microsoft.Recognizers.Text.DateTime/Utilities/TimexUtility.cs

```diff
@@ -457,6 +484,13 @@ public static string ModifyAmbiguousCenturyTimex(string timex)
             return "XX" + timex.Substring(2);
         }
 
+        public static float ParseNumberFromDurationTimex(string timex)
+        {
+            char[] unitChar = new char[] { 'D', 'W', 'M', 'Y', 'B' };
```

Comment:
This should either be static in class or in Constants.

Reply:
done in #2930